src/storage/query/GetEdgesStatisProcessor.cpp

```diff
@@ -0,0 +1,101 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "storage/query/GetEdgesStatisProcessor.h"
+#include "utils/IndexKeyUtils.h"
+
+namespace nebula {
+namespace storage {
+
+void GetEdgesStatisProcessor::process(const cpp2::GetEdgesStatisRequest& req) {
+    spaceId_ = req.get_space_id();
+    auto parts = req.get_parts();
```

Comment:
Query engine is responsible for `parts`? I suppose we need to collect all val in all owned partition. Can query engine get all partition of a storage owns?

Reply:
Please refer to `GraphStorageClient::GetEdgesStatis` interface

```
folly::SemiFuture<StorageRpcResponse<cpp2::GetStatisResponse>>
GraphStorageClient::getEdgesStatis(GraphSpaceID space,
                                   IndexID indexId,
                                   folly::EventBase* evb) {
```

So query engine does not need to know `parts`, storageclient handles it by itself。

One GetEdgesStatisRequest may handle multiple parts,
For the query engine, it only needs to process all returned reposes.